src/main/java/emissary/config/ServiceConfigGuide.java

```diff
@@ -829,12 +830,44 @@ public Map<String, String> findStringMatchMap(@Nullable final String theParamete
      */
     @Override
     public Map<String, Set<String>> findStringMatchMultiMap(@Nullable final String param) {
+        if (param == null) {
+            return Map.of();
+        }
+
+        return findStringMatchMultiMap(param, false);
+    }
+
+    /**
+     * Find entries beginning with the specified string and return a hash keyed on the remainder of the string with the
+     * value of the config line as the value of the hash Multiple values for the same hash are allowed and returned as a
+     * Set.
+     *
+     * <pre>
+     * {@code
+     * Example config entries
+     *    FOO_ONE: AAA
+     *    FOO_TWO: BBB
+     *    FOO_TWO: CCC
+     * Calling findStringMatchMap("FOO_",true)
+     * will yield a map with Sets
+     *     ONE -> {AAA}
+     *     TWO -> {BBB,CCC}
+     * }
+     * </pre>
```

Comment:
Thank you for this example, it helps a lot.

Reply:
> I assume that the default case (preserve order false) would mean that the TWO values could be in any arbitrary order depending on whether it was a Coordination place config laid over top.

Depends on how the key hashes get sorted, which isn't a sort order we can control with our config entries.